### PR TITLE
Add common problems page

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR adds a new page for the node docs with an overview of common problems and identifies which of the errors can be ignored, and which ones require attention. 

All information is taken from our Discord and Forum, provided solutions and responses are mostly Justin's and Nazar's comments. 

Please have a look and let me know if you see any errors worth adding to the list, of any incorrect solutions/responses. 

Thanks!

### CODE REVIEW

**Review by jim-counter — CHANGES_REQUESTED** [review]
We should not type the code blocks as it leads to keywords being highlighted that could cause confusion. I have only raised issues in the first copy of the page but they will need repeating for the extra version.

Otherwise, great initiative to help support our community!

File: docs/protocol/common_problems.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Most modern desktop processors starting with Broadwell on the Intel side and Ryzen (ZEN 1) on the AMD side do support necessary ADX instructions support and shouldn't be affected by the error.
+
+
+### No rewards after multiple days of farming
+
+
+Please make sure to:
+- Use our latest stable release, as shown on our [Github releases](https://github.com/subspace/subspace-cli/releases)
+- Verify your farmer is present and on the highest block on our [telemetry server](https://telemetry.subspace.network/)
+- Verify your balance using the [proper explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc-0.gemini-3e.subspace.network%2Fws#/explorer)
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/protocol/common_problems.md:55]: Proper Polkadot explorer xD

File: docs/protocol/common_problems.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+```
+
+
+Your PC likely ran out of space. Consider freeing up some space by removing unnecessary files, and then try again. Alternatively, you may adjust the plot amount to match the available disk space
+
+
+### Fast node synchronization (more than 100+ blocks per second) goes only up to ±20k blocks, then synchronization speed drops significantly.
+
+
+As the database gets fuller and blocks get bigger (as farmers started to produce votes), it is expected that the sync speed will settle on a smaller number. We have done some performance improvements in Gemini 3e and will do more performance tuning when the protocol is functionally complete.
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/protocol/common_problems.md:100]: That sounds better, thanks!

File: docs/protocol/common_problems.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+```bash
+0: Failed to build a farmer
+1: Single disk plot creation error: I/O error: File exists (os error 17)
+2: I/O error: File exists (os error 17)
+3: File exists (os error 17)
+```
+
+
+The system is detecting a pre-existing installation. If this is the case, you might consider wiping the current setup and re-initializing the CLI to ensure a clean installation.
```
[the remark this answers is not in the document]
Comment (EmilFattakhov) [reply docs/protocol/common_problems.md]: Good idea, added

**Review by jim-counter — APPROVED** [review]
LGTM. Thanks for the revisions @EmilFattakhov 👍

**Review by ImmaZoni — APPROVED** [review]
Looks great to me, thanks Emil! 🙏

**Pull request merged**